pytorch_lightning/callbacks/progress/base.py

```diff
@@ -213,11 +213,13 @@ def get_standard_metrics(trainer: "pl.Trainer", pl_module: "pl.LightningModule")
     if pl_module.truncated_bptt_steps > 0:
         items_dict["split_idx"] = trainer.fit_loop.split_idx
 
-    if trainer.logger is not None and trainer.logger.version is not None:
+    if len(trainer.loggers) == 1:
+        assert trainer.logger
         version = trainer.logger.version
-        if isinstance(version, str):
-            # show last 4 places of long version strings
-            version = version[-4:]
-        items_dict["v_num"] = version
+        if version is not None:
+            if isinstance(version, str):
+                # show last 4 places of long version strings
+                version = version[-4:]
+            items_dict["v_num"] = version
```

Comment:
we drop the version number for multiple loggers, which I find very reasonable considering we anyway only showed the ending characters.
not sure if it's worth mentioning in the changlog

Reply:
We could consider doing something similar to the suggestion [here](https://github.com/PyTorchLightning/pytorch-lightning/pull/11920#discussion_r810399237) for this. Keep the exact same behavior as a pure refactor, and then create an issue to discuss, which I am planning to do any way for logger v_num.